contributing.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+We're always looking to improve this project, open source contribution is encouraged so long as they adhere to these
+guidelines.
+
+## Pull Requests
+
+The Flagsmith team will be monitoring for pull requests. When we get one, a member of team will test the work against
+our internal uses and sign off on the changes. From here, we'll either merge the pull request or provide feedback
+suggesting the next steps.
+
+### A couple things to keep in mind
+
+- If you've changed APIs, update the documentation.
+- Keep the code style (indents, wrapping) consistent.
```

Comment:
I think you can just defer to `black` here?

Reply:
We also have JS YAML etc

Reply:
Ah yes, fair point.